components/readwise/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/readwise",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

As per [the semantic versioning guidelines](https://pipedream.com/docs/components/guidelines/#versioning), this should be a new `minor` version since a new component (feature) is being introduced.
```suggestion
  "version": "0.1.0",
```